decidim-accountability/app/views/decidim/accountability/admin/results/_form.html.erb

```diff
@@ -57,11 +57,9 @@
         </div>
       </div>
 
-    <% if Decidim::Accountability.enable_proposal_linking %>
```

Comment:
If you remove this, and the proposals module is not installed / enabled, this will actually throw an error as the partial path could not be found.

Reply:
How about replacing that `if ` with something like `if Decidim.module_installed?(:proposals)` ...

Reply:
How does this method work? @alecslupu in the edit of each module concerned, the option to pick a proposal in the form still exists? 

![image](https://github.com/user-attachments/assets/9ba4b169-b1cb-4eb1-835b-d97269adba86)